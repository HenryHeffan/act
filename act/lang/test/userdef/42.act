interface testme (bool x);


defproc testme2 (bool x) :> testme { x -> z }
{
  prs { 
    true -> x-
  }
}

testme2 t;

ptype (testme) t1;
