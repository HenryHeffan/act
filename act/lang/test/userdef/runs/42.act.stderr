ERROR: File `42.act', line 4, col 39
	`z' is not a port in process `testme2'
